lib/rrd_updates.ml

```diff
@@ -192,6 +192,19 @@ let create_multi prefixandrrds start interval cfopt =
        (fun (_prefix,rrd) ->
           (* Find the rrds that satisfy the requirements *)
           Rrd.find_best_rras rrd pdp_interval cfopt start) prefixandrrds) in
+  let first_rra =
+    match rras with
+    | []                              -> raise No_RRA_Available
+    | (xs::_) when List.length xs = 0 -> raise No_RRA_Available
+    | xs::_                           -> List.nth xs 0
```

Comment:
This can be simplifed by going deeper with the pattern-matching:

```
    match rras with
    | (x :: _) :: _ -> x
    | _ -> raise No_RRA_Available
```